zulip/zulip/__init__.py

```diff
@@ -491,6 +491,9 @@ def end_error_retry(succeeded):
                 if files:
                     kwargs['files'] = req_files
 
+                # Make long-polling api request. Set timeout to 90 sec as a balance
+                # between a low traffic rate and a reasonable latency time in case of
+                # a connection failure.
```

Comment:
We probably actually want the timeout to be like 30 if longpolling is false, and only 90 if longpolling is true.